build.gradle

```diff
@@ -14,8 +14,19 @@ dependencies {
     testImplementation platform('org.assertj:assertj-bom:3.25.1')
     testImplementation('org.junit.jupiter:junit-jupiter')
     testImplementation('org.assertj:assertj-core')
+
+    testImplementation ('org.mockito:mockito-core:5.0.0')
+    testImplementation ('org.mockito:mockito-junit-jupiter:5.0.0')
+
+    testImplementation 'org.springframework:spring-test:5.3.10'
 }
 
 test {
     useJUnitPlatform()
 }
+
+tasks.named('test') {
+    useJUnitPlatform()
+    jvmArgs '-Xshare:off' // JVM 아규먼트 설정
```

Comment:
무슨 이유로 추가하셨나요?

Reply:
테스트 코드 실행 시, OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended 라는 경고가 떠서 서치해보고 추가했습니다!